androiddemo/build.gradle.kts

```diff
@@ -34,12 +42,36 @@ android {
     }
 }
 
+kotlinter {
+    disabledRules = arrayOf("no-wildcard-imports")
+}
+
+val buildCrypto = task<Exec>("BuildCrypto") {
+    commandLine("${project.projectDir}/build_crypto.sh")
+}
+
+tasks.withType<Test>().configureEach {
+    dependsOn(buildCrypto)
+
+    doFirst {
+        val libsSharedPath = file("$buildDir/libs/shared").absolutePath
+        val pedersenPath = file("${rootDir}/crypto/pedersen/build/bindings").absolutePath
+        val poseidonPath = file("${rootDir}/crypto/poseidon/build/bindings").absolutePath
+        systemProperty("java.library.path", "$libsSharedPath:$pedersenPath:$poseidonPath")
+    }
+}
```

Comment:
Yes, it is for unit tests inside `androidemo`. 
We fetch deps from aar when running a demo, but unit tests are intended to run on host machine: https://github.com/software-mansion/starknet-jvm/blob/bd444b38416950a61853440151cc8afded315a99/androiddemo/src/test/java/com/example/androiddemo/ExampleUnitTest.kt#L7

We could also technically remove `androiddemo` unit tests altogether. Unlike demo app itself, they don't make much sense: basically doing the same thing as `lib` tests. 

But, if we should have these tests, then they at least should be able to work properly. And for that, crypto deps need to be built and available in path.

Reply:
yes, let's remove them then. this is just a demo, nothing very fancy and critical :+1:

Reply:
I moved `DepsTest` to instrumented tests. There, they make more sense since the deps are loaded from aar.

I also thought it might be better not to break [intended structure that contains unit tests](https://developer.android.com/studio/test/test-in-android-studio#test_types_and_locations), should anyone ever want to expand them, so I added some explanation in the comment.